src/perception/lidar_object_detection/lidar_object_detection/lidar_object_detection/lidar_object_detection_node.py

```diff
@@ -0,0 +1,211 @@
+# pylint: disable=wrong-import-position
+from sensor_msgs.msg import PointCloud2, PointField
+from vision_msgs.msg import ObjectHypothesisWithPose, Detection3D, Detection3DArray
+from visualization_msgs.msg import Marker, MarkerArray
+from pcdet.config import cfg, cfg_from_yaml_file
+from pcdet.datasets import DatasetTemplate
+from pcdet.models import build_network, load_data_to_gpu
+from pcdet.utils import common_utils
+import torch
+import numpy as np
+from rclpy.node import Node
+import rclpy
+import argparse
+import sys
+sys.path.append("/home/bolty/OpenPCDet")
```

Comment:
no absolute paths

Reply:
Hmm I remember this being required for the node to recognize openpcdet for some reason, any advice to get around it?